test/unit/tracks/video-track.test.js

```diff
@@ -0,0 +1,199 @@
+import VideoTrack from '../../../src/js/tracks/video-track';
+import * as VideoTrackEnums from '../../../src/js/tracks/video-track-enums';
+import TrackBaseline from './track-baseline';
+
+const defaultTech = {
+  videoTracks() {},
+  on() {},
+  off() {},
+  currentTime() {}
+};
+
+q.module('Video Track', {
+  beforeEach() {
+    this.videoTrackList = [];
+    Object.defineProperty(this.videoTrackList, 'selectedIndex', {
```

Comment:
this unit test was created before I had VideoTrackList, so I will use that now instead